x/community/keeper/staking.go

```diff
@@ -61,7 +61,13 @@ func (k Keeper) PayoutAccumulatedStakingRewards(ctx sdk.Context) {
 	// save state
 	k.SetStakingRewardsState(ctx, state)
 
-	return
+	// emit event
+	ctx.EventManager().EmitEvent(
```

Comment:
do we want to still emit events even if there are no payouts? or only when truncatedRewards is non-zero

Reply:
I'm split but leaning towards only emit if non-zero and we can move this into the if statement above.

Reply:
On deeper thought, ya let's only emit when we actually transfer funds